### ISSUE TITLE
fetch management canister interface from spec (or replica) instead of hardcoding it

### ISSUE THREAD

**Issue by letmejustputthishere** [dfinity/sdk#3595]
I tried the following:

I imported the management canister in a Motoko project like this
```
import Ic "ic:aaaaa-aa";
```

I expected to see this happen: The exposed types match the types in the interface specification

Instead, this happened: The compilation failed with the following error
```
/Users/moritz/projects/icrc7_launchpad/backend/factory.mo:8.36-8.53: type error [M0030], type field install_code_args does not exist in type
```

### Meta

`dfx --version`:
```
0.17.0
```

<details><summary>Further log output</summary>
<p>

```
<log output>
```

</p>
</details>

**Comment by sesi200** [comment]
I agree this would be nice to have, but it's a low priority maintenance task for now.

Tracked internally in [this ticket](https://dfinity.atlassian.net/browse/SDKTG-269)
